andela/casablanca-cfh
# #156096144 Configure ESLint

#### What does this PR do?
This PR configures ESLint.
#### Description of Task to be completed?
Configuring ESLint for the project. The purpose of this is to check style violations across the project and flag them.
#### How should this be manually tested?

- clone the repo
-  cd into the project directory
- run `npm install` to install dependencies
- run `npm run lint **` to see style violations across the project

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?(if applicable)
The pivotal tracker Id for this task is `#156096144`
